src/entities/Email.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Could you please suggest if this eslint disable is still necessary?🙂

Reply:
it was necessary because of EsLint bug: decorators were treated as 'not used'. So I updated packages versions, and not it's resolved 🫣